cmd/tools/vcreate/tests/new_with_model_arg.expect

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/expect
+
+set timeout 1
+
+# Pass v_root as arg, since we chdir into a temp directoy during testing and create a project there.
+set v_root [lindex $argv 0]
+set project_name [lindex $argv 1]
+set model [lindex $argv 2]
+
+spawn $v_root/v run $v_root/cmd/tools/vcreate/ new $project_name $model
```

Comment:
```suggestion
spawn $v_root/v new $project_name $model
```

Reply:
This will be closer to what the final user will type/run . V should take care of compiling its `cmd/tools/vcreate` and then run it by itself.

Reply:
That's certainly closer to the actual usage.
It seems that using `v new` for the compiled v executable doesn't want to work yet on mac. Mac wasn't failing when running `vcreate.v` directly.